renv.lock

```diff
@@ -1335,9 +1335,9 @@
     },
     "tibble": {
       "Package": "tibble",
-      "Version": "3.1.8",
+      "Version": "3.2.1",
       "Source": "Repository",
-      "Repository": "CRAN",
+      "Repository": "RSPM",
```

Comment:
Is this right, or do we want to point to CRAN instead?

Reply:
RSPM is fine, it's just the pre-built posit linux binaries.